src/validateEmail.test.js

```diff
@@ -17,4 +17,138 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   // write more tests here
+  
+  it('should return "false" non-english letters', () => {
+   expect(validateEmail('bąk@doom.com.'))
+    .toBe(false);
+  
+});
+
+it('should return "true" for digits in email', () => {
+  expect(validateEmail('email123@mail123.com123.'))
+   .toBeTruthy();
+ 
+});
+
+it('should return "true" for only digits email.', () => {
+  expect(validateEmail('12424@342.234'))
```

Comment:
The email '12424@342.234' is considered invalid because the domain part is numeric. The expectation should be `toBeFalsy()` instead of `toBeTruthy()`.

Reply:
code accepts it